### ISSUE TITLE
add jobdsl support for locale multi selection text field

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/pipeline-ce#143]
we already have something similar for groups. let's copy approach and make support for "jenkinsLocale"
```
                //** Requires Active Choices Plug-in v1.2+ **//*
                //** Currently renders with error: https://issues.jenkins-ci.org/browse/JENKINS-42655 **//*
                if (!isParamEmpty(currentSuite.getParameter("jenkinsGroups"))) {
                    activeChoiceParam("groups") {
                        description("Please select test group(s) to run")
                        filterable()
                        choiceType("MULTI_SELECT")
                        groovyScript {
                            script(this.listToString(currentSuite, "jenkinsGroups"))
                            fallbackScript("return ['error']")
                        }
                    }
                }
```

**Comment by vdelendik** [comment]
done. during the testing please declare in any your TestNG suite xml file jenkinsLocale parameter with comma separated lists of locales:
```
	<parameter name="jenkinsLocale" value="en_US,ca_CA,fr_FR"/>
```

After scanning using this updated repo/branch make sure to see multi-selection locale field, i.e. you could choose several locales at ones and run jobs.
Note: job itself might fail as carina not support multi locales as of now

**Comment by vdelendik** [comment]
fixed in 1.5.2 pipeline-ce version.
one major note for the testing exists!
First time during the scaning the repo with `<parameter name="jenkinsLocale" value="en_US,ca_CA,fr_FR"/>` we expect to see the failure as it requires extra scripts permissions.
So after register repository failure goto:
1. manage jenkins -> in-process script approval
2. approve 2 commands like `return ["en-US", ...]` and `return ['error']`
3. register repository again
-> job should be green, adjusted suite should have multi-selected choice `locale`

**Comment by okamara** [comment]
Verified on zebrunner ce from dev. branch. Fixed.
<img width="320" alt="Снимок экрана 2021-06-18 в 13 00 26" src="https://user-images.githubusercontent.com/32964663/122544182-37074700-d035-11eb-8f02-79df3ad6a395.png">
<img width="1216" alt="Снимок экрана 2021-06-18 в 13 00 10" src="https://user-images.githubusercontent.com/32964663/122544186-3969a100-d035-11eb-8072-603f8b5b6ef3.png">
